corehq/apps/export/static/export/js/download_export.js

```diff
@@ -21,4 +21,8 @@ hqDefine('export/js/download_export', function() {
             return $('#id_user_types');
         },
     });
+
+    $(function() {
```

Comment:
if the initial page data gets above works without being in a document ready handler, does it need to be here?

Reply:
offlined, going to leave this alone because moving it disturbs angular